pkg/libxdploader/README.md

```diff
@@ -0,0 +1,14 @@
+# Install libxdp headers
+```shell
+ git clone https://github.com/xdp-project/xdp-tools.git
+ cd xdp-tools
```

Comment:
We've got all of that stuff already packaged and ready to pull with yum (talking about something being supported, we support our own builds of libxdp and libbpf, and not builds from source):
~~~
yum install libxdp libxdp-devel xdp-tools libbpf libbpf-devel
~~~

I don't remember what rpm was required where, but we defo don't have to rebuild anything